README.md

```diff
@@ -8,6 +8,39 @@ effectively together with [nginx's autoindex module](http://nginx.org/en/docs/ht
 
 A sample nginx configuration is also included which mounts **file browser** under root (`/`) and mounts files to be listed under `/files` path. Hence is the `filesBaseUrl` under
 
+## Installation / Configuration
+
+The system has a default to use the location's root (E.G. the site's root location `www.example.net/`, and the physical of `/opt/www/`). This can be configured to move both the ***installation*** location and the location to be browseable.
+
+1. Clone the repo to the server
+2. Create an installation location (E.G. `/var/www/files/`)
+3. Copy the `css`, `image`, `js` directories to the installation location
+4. Copy the `index.html` to the installation location
+5. Create the location for the files to be shared & browsable (E.G. `/var/www/isos`)
+6. Copy the files to the location
+7. Make sure the files in the installation location & the location for the files to be shared are owned by the nginx process (E.G. `www-data:www-data` for Debian/Ubuntu based saystems)
```

Comment:
is owning required? I think being able to read files and list the directories should be enough.

Reply:
It only must be _readable_ by `www-data`, being _owned_ by it is overkill. Maybe this can be worded as:
```markdown
7. Make sure both the UI and shared files and directories are at least readable by the nginx process[...] with `chmod +x -R ...`. Or `www-data` own the files and directories: `chown -R www-data: ...`
```

Reply:
this is definitely better, we just need someone to update the PR :)